lib/newgistics/configuration.rb

```diff
@@ -19,5 +19,41 @@ def time_zone=(name)
     def local_time_zone=(name)
       @local_time_zone = TimeZone.new(name)
     end
+
+    def logger
+      @logger ||= DefaultLogger.build
+    end
+
+    def logger=(_logger)
+      @logger = _logger
+    end
+
+    def log_http
+      @log_http || false
+    end
+
+    def log_http_headers=(value)
```

Comment:
Can you explain this pattern to me? It's new.

```ruby
def test
  ...
end

def test=(a)
  ...
end

Reply:
@Jetbo It overrides the assignment operator. This defines the method that executes when you do something like

```ruby
Newgistics.configuration.test = 123
```